src/HttpRequestInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BEAR\Resource;
+
+/**
+ * Sends a HTTP request using cURL
+ */
+interface HttpRequestInterface
+{
+    /**
+     * Sends a HTTP request using cURL
```

Comment:
```suggestion
     * Sends a HTTP request
```

interface なので、cURL に関する情報は不要ですかね？